infrastructure/local/docker-compose-full.yml

```diff
@@ -61,8 +61,10 @@ services:
       - "8088:80"
     env_file:
       - ../../.env-ui
+      - ../../.env-issuer
```

Comment:
what about ../ui/.env? Does that need to be loaded too? When I follow the README, this is what I see:
![image](https://github.com/user-attachments/assets/54499714-f93b-4fce-987c-cda5ff81a746)

Reply:
The `./ui/.env` file is needed only when you run the fe as a developer, executing `npm run ...`. It is not needed when you run the issuer node with docker. 
Regarding your problem, which steps are you following to run the issuer node?

Reply:
Ok. Never mind. I did finally get the issuer node running locally yesterday, but it was difficult to figure our the nginx and ngrok configuration needed to get past the CORS errors.